torch/_dynamo/external_utils.py

```diff
@@ -3,6 +3,8 @@
 import functools
 from typing import List
 
+from typing_extensions import deprecated  # Python 3.13+
```

Comment:
What does the comment mean here?

Reply:
This means the `deprecated` API is introduced in Python 3.13 stdlib. We can replace the `typing_extensions` import when we drop Python 3.12 support.

Reply:
Ruff will replace all these imports for us when we update the minimum Python version high enough.